src/applications/mhv/secure-messaging/containers/MessageDetails.jsx

```diff
@@ -75,7 +75,7 @@ const MessageDetail = () => {
   let pageTitle;
 
   if (isSent) {
-    pageTitle = 'Sent messages';
+    pageTitle = 'Sent';

```

Comment:
Since we're only using the header name now you can't get rid of this.  We originally put this here to accommodate the unique header name.